modules/analyse/src/main/AnalyseBsonHandlers.scala

```diff
@@ -26,7 +28,7 @@ object AnalyseBsonHandlers:
         startPly = startPly,
         date = r.date("date"),
         fk = r.strO("fk"),
```

Comment:
note that "npm" was moved inside the engine doc in a soft migration. you might want to move "fk" in there as well. i left it alone in case that field's location is relied on by external scripts.

the "engine" subdoc within db.analysis2 is mostly separate for aesthetic reasons and to leverage default db handlers within its container (which defines explicit reads and writes). but i could flatten those Engine fields directly into Analysis if you'd prefer.

Reply:
In fact I think we should just drop `fk` and rely on `engine.userId` for fishnet as well